LibraBFT/Impl/Handle.agda

```diff
@@ -102,46 +102,33 @@ module LibraBFT.Impl.Handle where
  ...| V v = processVote now v
  ...| C c = return unit            -- We don't do anything with commit messages, they are just for defining Correctness.
 
+ -- Translation to SystemModel
+ import LibraBFT.Yasm.Types as LYT
  -- For now, the SystemModel supports only one kind of action: to send a Message.  Later it might
- -- include things like logging, crashes, assertion failures, etc.  At that point, definitions like
- -- the following might become part of the SystemModel, but they are included here to enable the
- -- temporary scaffolding below.
-
- data Action (Msg : Set) : Set where
-   send : Msg → Action Msg
-
- action-send-injective : ∀ {Msg}{m m' : Msg} → send m ≡ send m' → m ≡ m'
- action-send-injective refl = refl
-
- msgToSend : {Msg : Set} → Action Msg → Msg
- msgToSend (send m) = m
-
- msgToSend≡ : ∀ {Msg x}{m : Msg} → m ≡ msgToSend x → send m ≡ x
- msgToSend≡ {_} {send m} {m} refl = refl
+ -- include things like logging, crashes, assertion failures, etc.
```

Comment:
This comment is similarly unclear/unnecessary.  It reads like a TODO, but it's not one.  It seems more of a comment on what's changing than what remains here.

Reply:
This is an edited version of a comment that was here before the changes. I realize now it doesn't belong here (there's already talk of desirable future actions in `LibraBFT.Yasm.Types`), so I'll remove it.